documentation/developer/language/04_arrays_and_tuples.md

```diff
@@ -72,6 +72,30 @@ Leo supports static arrays with fixed size.
 **Dynamic arrays do not exist in Leo**. 
 You cannot change the length of an array once it is initialized.
 
+### Arrays without size 
```

Comment:
Arrays without size => Arrays with Unspecified Size

(Reason: arrays always have a size, we just allow it to be left unspecified.)